ant-protocol/src/storage/chunks.rs

```diff
@@ -25,6 +25,9 @@ pub struct Chunk {
 }
 
 impl Chunk {
+    /// The maximum size of a chunk is 1MB
+    pub const MAX_SIZE: usize = 1024 * 1024;
```

Comment:
What is the relation between this and the `self_encryption::MAX_CHUNK_SIZE`? Should they not be the same?

Reply:
They should be equal but the alternative to this constant here is to add self_encryption as a new dependency to sn_protocol which I didn't want to

Reply:
given our current code aiming to have max 4MB, specify a const here with different vallue is bit risky?
or, it needs to be something like: DEFAULT_MAX_SIZE ?

Reply:
Can we make this configurable? We're currently setting chunk size to 4MiB on CI.

Reply:
It would be better to have only one place were this constant exists and have it be configurable. 
For this reason maybe using the one in self-encryption is best after all... Will patch that for now.